include/opt-sched/Scheduler/aco.h

```diff
@@ -45,7 +52,9 @@ class ACOScheduler : public ConstrainedScheduler {
   pheromone_t &Pheromone(SchedInstruction *from, SchedInstruction *to);
   pheromone_t &Pheromone(InstCount from, InstCount to);
   double Score(SchedInstruction *from, Choice choice);
-  bool shouldReplaceSchedule(InstSchedule *OldSched, InstSchedule *NewSched);
+  bool shouldReplaceSchedule(InstSchedule *OldSched, InstSchedule *NewSched,
+                             bool IsGlobal);
+  DCF_OPT ParseDCFOpt(std::string opt);
```

Comment:
I'd use `llvm::StringRef` or `const std::string&` instead of `std::string`, which means that the string won't need to be copied into the argument of `ParseDCFOpt()`

Reply:
will do